x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecks.java

```diff
@@ -595,4 +595,40 @@ static DeprecationIssue checkClusterRoutingAllocationIncludeRelocationsSetting(f
             DeprecationIssue.Level.CRITICAL
         );
     }
+
+    static DeprecationIssue checkFixedAutoQueueSizeThreadpool(final Settings settings,
```

Comment:
Should this function be added to the `DeprecationChecks` file?

Reply:
Oh wow yeah definitely. Thanks for catching it. Done.